privateca/snippets/noxfile_config.py

```diff
@@ -30,8 +30,8 @@
     # to 'BUILD_SPECIFIC_GCLOUD_PROJECT' if you want to opt in using a
     # build specific Cloud project. You can also use your own string
     # to use your own Cloud project.
-    # "gcloud_project_env": "GOOGLE_CLOUD_PROJECT",
-    "gcloud_project_env": "BUILD_SPECIFIC_GCLOUD_PROJECT",
+    "gcloud_project_env": "GOOGLE_CLOUD_PROJECT",
```

Comment:
This will definitely cause quota issues as privateCA has a low quota of only 5 CAs created per minute in a single project, which can't be changed. We are already running into this issue even with separate build projects.